shavar/views/__init__.py

```diff
@@ -47,9 +47,21 @@ def list_view(request):
 
 
 def downloads_view(request):
-    resp_payload = {'interval': _setting(request, 'shavar', 'default_interval',
-                                         45 * 60),
-                    'lists': {}}
+
+    # Use the new config variable name but support the old one for
+    default_interval = _setting(request, 'shavar', 'default_interval', None)
+    backoff_delay = _setting(request, 'shavar', 'client_backoff_delay', None)
+
+    # Throw a fit if both are specified
+    if default_interval is not None and backoff_delay is not None:
+        raise ConfigurationError("Specify either default_interval or "
+                                 "client_backoff_delay in the [shavar] "
+                                 "section of your config but not both.\n"
+                                 "client_backoff_delay is preferred.")
+
+    delay = backoff_delay or default_interval or 45 * 60
```

Comment:
To be closer to the spec, it may be worthwhile to have 30 minutes as the default in the code. The 45 minute default is kind of weird and will likely lead to unpredictable behavior.

@ckolos is right, I suggest we configure both stage and prod to use a default polling interval of 60 minutes. That doesn't have to be in code though, it could be done via `default_interval` in the config.